cmd/frontend/graphqlbackend/search_test.go

```diff
@@ -342,3 +343,52 @@ func Test_detectSearchType(t *testing.T) {
 		})
 	}
 }
+
+func Test_QuoteSuggestions(t *testing.T) {
+	t.Run("regex error", func(t *testing.T) {
+		raw := "*"
+		_, alert := processQuery(raw, query.SearchTypeRegex)
+		if alert == nil {
+			t.Fatalf(`alert returned from syntax.Parse("%s") is nil`, raw)
```

Comment:
Alternatively if you want to quota a string/integer in a format string:
```suggestion
			t.Fatalf("alert returned from syntax.Parse(%q") is nil", raw)
```

Reply:
This is just a copy paste from existing tests. I'll prefer to do format/clean tests in separate PRs.